test/utils/utils.test.js

```diff
@@ -51,7 +51,25 @@ describe('Utils', () => {
         expect(link.target).to.equal('_blank');
       });
     });
-  
+    
+    describe('Configure Auto Block', () => {
+      it('Disable auto block when #_dnb in url', () => {
+        const disableAutoBlockLink =
+          document.querySelector('.disable-autoblock');
+        utils.decorateLinks(disableAutoBlockLink);
+        expect(disableAutoBlockLink.href).to.equal(
+          'https://www.instagram.com/'
+        );
+      });
+
+      it('Auto block works as expected when #_dnb in not added to url', () => {
```

Comment:
Nit, typo: should be `is`, not `in`.

Reply:
Thanks for noticing this @honstar this is fixed now.